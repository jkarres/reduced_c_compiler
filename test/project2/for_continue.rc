
function : void main() {
  int i, j;

  for (i = 0; i < 10; ++i) {

    cout << "i is now " << i << endl;

    for (j = 0; j < 10; ++j) {

      cout << "  j is now " << j << endl;

      if (i * j % 2 == 0) {
        continue;
      }

      cout << "    " << i * j << " is odd" << endl;

    }

  }

}
