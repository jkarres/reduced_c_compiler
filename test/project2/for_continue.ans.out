i is now 0
  j is now 0
  j is now 1
  j is now 2
  j is now 3
  j is now 4
  j is now 5
  j is now 6
  j is now 7
  j is now 8
  j is now 9
i is now 1
  j is now 0
  j is now 1
    1 is odd
  j is now 2
  j is now 3
    3 is odd
  j is now 4
  j is now 5
    5 is odd
  j is now 6
  j is now 7
    7 is odd
  j is now 8
  j is now 9
    9 is odd
i is now 2
  j is now 0
  j is now 1
  j is now 2
  j is now 3
  j is now 4
  j is now 5
  j is now 6
  j is now 7
  j is now 8
  j is now 9
i is now 3
  j is now 0
  j is now 1
    3 is odd
  j is now 2
  j is now 3
    9 is odd
  j is now 4
  j is now 5
    15 is odd
  j is now 6
  j is now 7
    21 is odd
  j is now 8
  j is now 9
    27 is odd
i is now 4
  j is now 0
  j is now 1
  j is now 2
  j is now 3
  j is now 4
  j is now 5
  j is now 6
  j is now 7
  j is now 8
  j is now 9
i is now 5
  j is now 0
  j is now 1
    5 is odd
  j is now 2
  j is now 3
    15 is odd
  j is now 4
  j is now 5
    25 is odd
  j is now 6
  j is now 7
    35 is odd
  j is now 8
  j is now 9
    45 is odd
i is now 6
  j is now 0
  j is now 1
  j is now 2
  j is now 3
  j is now 4
  j is now 5
  j is now 6
  j is now 7
  j is now 8
  j is now 9
i is now 7
  j is now 0
  j is now 1
    7 is odd
  j is now 2
  j is now 3
    21 is odd
  j is now 4
  j is now 5
    35 is odd
  j is now 6
  j is now 7
    49 is odd
  j is now 8
  j is now 9
    63 is odd
i is now 8
  j is now 0
  j is now 1
  j is now 2
  j is now 3
  j is now 4
  j is now 5
  j is now 6
  j is now 7
  j is now 8
  j is now 9
i is now 9
  j is now 0
  j is now 1
    9 is odd
  j is now 2
  j is now 3
    27 is odd
  j is now 4
  j is now 5
    45 is odd
  j is now 6
  j is now 7
    63 is odd
  j is now 8
  j is now 9
    81 is odd
